build/source/driver/multi_driver.f90

```diff
@@ -266,7 +288,7 @@ program multi_driver
 ! nGRU-is the total number of GRUs of the simulation domain
 ! nHRU-is the total number of HRUs of the simulation domain
 ! hruCount-is a local variable for the total number of HRUs in a GRU
-call allocate_gru_struc(nGRU,nHRU,err,message); call handle_err(err,message)
+call allocate_gru_struc(nGRU,nHRU,strtHRU,err,message); call handle_err(err,message)
```

Comment:
We do need the start index to allocate, since this determines the number of GRUs (or HRUs).
startHRU = intent(in)
nHRU = intent(inout) -- intent(in) if using basic parallelization; intent(out) if full simulation
nGRU = intent(out) -- the GRUs associated with the desired HRUs

(note that I'm suggesting to modify the parallelization to be based on GRUs, where `nGRU` would be `intent(inout)`)